SPECS/curl/curl.spec

```diff
@@ -88,6 +90,10 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %{_libdir}/libcurl.so.4*
 
 %changelog
+* Thu Jan 12 2023 Aurélien Bombo <abombo@microsoft.com> - 7.86.0-2
+- Downgrade to 7.86.0 since 7.87.0 breaks perl-WWW-curl.
+- Apply patch to fix CVE-2022-43551.
+
 * Tue Jan 10 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 7.87.0-1
```

Comment:
I think we should clear this entry from the changelog to avoid potential tooling issues (out of order version numbers)